cloudbuild.yaml

```diff
@@ -1,4 +1,9 @@
 steps:
+- name: gcr.io/cloud-builders/docker
+  entrypoint: 'bash'
+  args: ['-c', 'rm -rf .* * || true']
```

Comment:
Nice hack! I remember this being a blocker for use of LFS for something in the past